### PR TITLE
Test the Chrome web browser host in automation

### PR DESCRIPTION
**Pull request by jugglinmike**

### CODE REVIEW

**Comment by jugglinmike** [comment]
@bterlson Any thoughts on this?

File: runtimes/browser.js [hunk 1]
```diff
@@ -20,6 +20,12 @@ var $ = window.$ = {
     var fwin = frame.contentWindow;
     var fdoc = fwin.document;
     var fscript = fdoc.createElement('script');
+
+    // The following is a workaround for a bug in Chromium related to reporting
+    // errors produced from evaluating code using `eval`.
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=746564
+    fdoc.write('<body>');
```
Comment (rwaldron) [inline runtimes/browser.js]: I love this

Comment (jugglinmike) [inline runtimes/browser.js]: :D Believe it or not, it breaks Edge. Edge is currently not working anyway, though, so I will address that later.

Comment (rwaldron) [inline runtimes/browser.js]: I'm glad you reminded me of that... unfortunately, it also means that I won't land this until @bterlson has a chance to weigh in.

Comment (jugglinmike) [inline runtimes/browser.js]: Aw, shucks.

**Pull request merged**